core/controllers/classroom_test.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for the classroom page."""
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+
+from constants import constants
+from core.tests import test_utils
+import feconf
+
+
+class BaseClassroomControllerTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        """Completes the sign-up process for the various users."""
+        super(BaseClassroomControllerTests, self).setUp()
+        self.signup(self.NEW_USER_EMAIL, self.NEW_USER_USERNAME)
+        self.user_id = self.get_user_id_from_email(self.NEW_USER_EMAIL)
+
+
+class ClassroomPageTests(BaseClassroomControllerTests):
+
+    def test_any_user_can_access_classroom_page(self):
+        with self.swap(constants, 'ENABLE_NEW_STRUCTURE_PLAYERS', True):
+            self.login(self.NEW_USER_EMAIL)
+            self.get_html_response(
+                '%s/%s' % (feconf.CLASSROOM_URL_PREFIX, 'Math'))
```

Comment:
Worth checking for something in the response body that confirms it's the classroom page?

Reply:
But I'm not sending back any values, just the HTML and on printing out the reponse from `get_html_response`, it seemed like a generic HTML file.

Reply:
Done